apps/api/src/app/shared/framework/swagger/swagger.controller.ts

```diff
@@ -21,37 +21,37 @@ const options = new DocumentBuilder()
   .addTag(
     'Events',
     `Events represent a change in state of a subscriber. They are used to trigger workflows, and enable you to send notifications to subscribers based on their actions.`,
-    { url: 'https://docs.novu.co/workflows' }
+    { url: 'https://docs.novu.co/workflows?utm_campaign=inapp-swagger' }
```

Comment:
Per the comment above, let's revert all of these changes on the Swagger controller.

```suggestion
    { url: 'https://docs.novu.co/workflows' }
```